### ISSUE TITLE
ConfigServer Security & Firewall: UI update for 9.25+

### ISSUE THREAD

**Issue by qooob** [qooob/authentic-theme#568]
Hi,

_ConfigServer Security & Firewall_ team keeps updating _UI_ bit by bit constantly, breaking module support time after time on the theme side.

For those who installed _Authentic Theme_ 18.20 few days ago, it's recommended to manually reinstall (update) it right now, in order to get precise execution for the latest (9.27 at the moment) version of the module.

Other than that, you will also have to run manually from the _console_ `csf -u` - to update the module to the latest version, due to incompatible changes done on the module's side.

**Comment by qooob** [comment]
Third wave of updating _Authentic Theme_ 18.20 for better _ConfigServer Security & Firewall_ support.

Includes:
   1. Native configuration and back buttons in the module's header;
   2. Colorful buttons with icons on the index page;
   3. Minor improvements;

**Comment by hrace009** [comment]
when i try to click all this button, it won't show anything
![capture1](https://cloud.githubusercontent.com/assets/7691118/19894047/0deae0de-a07e-11e6-8196-466ab6d69378.JPG)

**Comment by qooob** [comment]
Maybe because you have `Test Mode` on in CSF settings. Other than that make sure that you have latest version of CSF installed. You can force update by running `csf -u`.

Also it's worth clearing browser's cache.

**Comment by hrace009** [comment]
my csf is the latest version, maybe because in testing mode, i will try to configure it since this is new server
<pre>[root@mysqldb csf]# csf -u
csf is already at the latest version: v9.27
*WARNING* TESTING mode is enabled - do not forget to disable it in the configuration</pre>

**Comment by qooob** [comment]
Yes, that's the case.

**Comment by SteveHawes** [comment]
I have two servers with the theme installed but on one of them the Firewall Configuration button is not showing and I have to edit the csf.conf file by hand.

Both have CSF V9.27 and but one is CentOS 7 and the other is CentOS 6 - any ideas?

Non-working one:
![no-button](https://cloud.githubusercontent.com/assets/23415735/20236913/5f430bb4-a8bc-11e6-93d5-b411e8368672.png)

Working one:
![with-button](https://cloud.githubusercontent.com/assets/23415735/20236927/ae4124da-a8bc-11e6-8c6b-08e85f0fbf4b.png)

Thanks
Steve

**Comment by qooob** [comment]
Look in the table's header. It's default for Webmin.

**Comment by SteveHawes** [comment]
Doh! - never thought of that one as I am so used to seeing the button in the list. Clicking the cog in the header works no problem but any idea why there is a difference between the two servers?

Thanks for the super-quick response

**Comment by qooob** [comment]
Just redownload the theme for that server.

**Comment by cloud4g** [comment]
This is an old topic, however, I made it a problem by turning the 'check for updates' in the Authentic theme to off on three servers.  The problem appeared a few weeks ago when CSF was upgraded.  I could not figure out how to fix it so used the command line and text edits of the config file until I realized Doh! that my version of the theme was out of date.  The update fixed it and I've reset the option to check for updates.
